core/test_network/src/lib.rs

```diff
@@ -371,10 +337,11 @@ impl Network {
 /// # Panics
 /// When unsuccessful after `MAX_RETRIES`.
 pub fn wait_for_genesis_committed(clients: &[Client], offline_peers: u32) {
-    const POLL_PERIOD: Duration = Duration::from_millis(1000);
-    const MAX_RETRIES: u32 = 10;
+    const POLL_PERIOD: Duration = Duration::from_millis(5000);
+    const MAX_RETRIES: u32 = 6;
```

Comment:
These numbers seem arbitrary. We need to have a clear idea on how to set them, and how to measure the timeouts for the types of deployment. Perhaps they should even be extracted into configuration files.